### PR TITLE
Update the description of Thunderbird.

### PR DESCRIPTION
**Pull request by clokep**
Based on much of the recent work that's happened for Matrix support in Thunderbird (and just the passage of time).

I also updated some incorrect values and expanded the description.

I see there's now a `gatsby/content/projects/clients` directory too, not sure if this should be moved into that?

CC @freaktechnik

Preview: https://pr1059--matrix-org-previews.netlify.app

### CODE REVIEW

File: gatsby/content/projects/2017-05-02-thunderbird.mdx [hunk 1]
```diff
@@ -3,14 +3,34 @@ layout: projectimage
...
 ---
 
-Initial support for Matrix in [Mozilla Thunderbird](https://www.mozilla.org/en-US/thunderbird/), available in Nightly since March 2017, and due to land in Instantbird/Thunderbird 54.
+[Thunderbird](https://www.thunderbird.net/) is a free, open-source email, calendar, and chat application which runs on multiple platforms. It supports multiple instant messaging protocols, including IRC, XMPP, and Matrix.
 
-To enable, turn `chat.prpls.prpl-matrix.disable` to `false` in the advanced config editor (>Tools >Options >Advanced >Config Editor).
+Thunderbird is [available for download](https://www.thunderbird.net/) and the [source code is available](https://hg.mozilla.org/comm-central). Additionally there is [documentation for getting started with contributing](https://developer.thunderbird.net/). Please [file any bugs found](https://bugzilla.mozilla.org/enter_bug.cgi?product=Chat+Core&component=Matrix).
+
+Matrix support in Thunderbird is enabled by default in Nightly builds (as of Thunderbird 91), but can be enabled in Beta and ESR builds by setting `chat.prpls.prpl-matrix.disable` to `false` in the advanced config editor (>Tools >Options >Advanced >Config Editor).
```
Comment (freaktechnik) [inline gatsby/content/projects/2017-05-02-thunderbird.mdx:36]: I think nightly is now consistently branded as "Daily".

**Comment by clokep** [comment]
(@afranke I think you're now the proper person to request review of for stuff like this, but not 100% sure!)

File: gatsby/content/projects/2017-05-02-thunderbird.mdx [hunk 2]
```diff
@@ -3,14 +3,34 @@ layout: projectimage
...
 license: MPL-2.0
-repo: 
+platforms:
+    - Linux
+    - Mac
+    - Windows
+repo: https://hg.mozilla.org/comm-central
+home: https://www.thunderbird.net/
+room: "#maildev:mozilla.org"
+features:
```
Comment (afranke) [inline gatsby/content/projects/2017-05-02-thunderbird.mdx:18]: Where did you copy this feature list from? It is not as populated as the one for e.g. Fractal and Thunderbird doesn’t show up in my local build in the [clients matrix](https://matrix.org/clients-matrix) so I’m guessing there isn’t enough information here.

File: gatsby/content/projects/2017-05-02-thunderbird.mdx [hunk 3]
```diff
@@ -3,14 +3,34 @@ layout: projectimage
...
 ---
 
-Initial support for Matrix in [Mozilla Thunderbird](https://www.mozilla.org/en-US/thunderbird/), available in Nightly since March 2017, and due to land in Instantbird/Thunderbird 54.
+[Thunderbird](https://www.thunderbird.net/) is a free, open-source email, calendar, and chat application which runs on multiple platforms. It supports multiple instant messaging protocols, including IRC, XMPP, and Matrix.
 
-To enable, turn `chat.prpls.prpl-matrix.disable` to `false` in the advanced config editor (>Tools >Options >Advanced >Config Editor).
+Thunderbird is [available for download](https://www.thunderbird.net/) and the [source code is available](https://hg.mozilla.org/comm-central). Additionally there is [documentation for getting started with contributing](https://developer.thunderbird.net/). Please [file any bugs found](https://bugzilla.mozilla.org/enter_bug.cgi?product=Chat+Core&component=Matrix).
+
+Matrix support in Thunderbird is enabled by default in Daily builds (as of Thunderbird 91), but can be enabled in Beta and ESR builds by setting `chat.prpls.prpl-matrix.disable` to `false` in the advanced config editor (>Tools >Options >Advanced >Config Editor).
```
Comment (afranke) [inline gatsby/content/projects/2017-05-02-thunderbird.mdx:36]: Are all stable builds ESR in Thunderbird? If not, how come Matrix can be enabled in ESR but not in stable?

Comment (freaktechnik) [reply gatsby/content/projects/2017-05-02-thunderbird.mdx:36]: Thunderbird stable builds are ESR, I guess we call it "release channel" on our website though.

File: gatsby/content/projects/2017-05-02-thunderbird.mdx (hunk 2 above)
Comment (clokep) [reply gatsby/content/projects/2017-05-02-thunderbird.mdx:18]: > Where did you copy this feature list from? It is not as populated as the one for e.g. Fractal and Thunderbird doesn’t show up in my local build in the [clients matrix](https://matrix.org/clients-matrix) so I’m guessing there isn’t enough information here.

I copied it from `element` and then removed the features that aren't supported. (It is super unclear what the options are?)

Comment (clokep) [reply gatsby/content/projects/2017-05-02-thunderbird.mdx:18]: > Thunderbird doesn’t show up in my local build in the [clients matrix](https://matrix.org/clients-matrix) so I’m guessing there isn’t enough information here.

I think the only things showing up there are the ones moved to `gatsby/content/projects/clients`? I'm not sure though.

I was pointed to https://matrix.org/docs/projects/client/mozilla-thunderbird directly and noted it was very out-of-date.

Comment (afranke) [reply gatsby/content/projects/2017-05-02-thunderbird.mdx:18]: Ha, could be. @benparsons can you confirm? Maybe this update would be a good opportunity to move this to the subdirectory.

File: gatsby/content/projects/2017-05-02-thunderbird.mdx (hunk 3 above)
Comment (afranke) [reply gatsby/content/projects/2017-05-02-thunderbird.mdx:36]: I’d welcome a rephrase that makes it clearer. I don’t expect regular users to know that ESR is what they get when they just head to the Thunderbird website and hit Download.

File: gatsby/content/projects/2017-05-02-thunderbird.mdx [hunk 4]
```diff
@@ -3,14 +3,34 @@ layout: projectimage
 title: Mozilla Thunderbird
 categories: 
  - client
-description: A free email & chat app
-author: Frédéric Wang
+description: A free email, calendar & chat app
+author: Mozilla
 maturity: Beta
 language: C
```
Comment (KitsuneRal) [inline gatsby/content/projects/2017-05-02-thunderbird.mdx:9]: The source code doesn't look like C - rather, JavaScript?

Comment (freaktechnik) [reply gatsby/content/projects/2017-05-02-thunderbird.mdx:9]: Well, depends on what part of the source, the Matrix specific part is indeed mostly implemented in JS

Comment (clokep) [reply gatsby/content/projects/2017-05-02-thunderbird.mdx:9]: This really should list half a dozen languages, but that doesn't seem possible. I think by core LOC (which is a useless measure, but whatever) "C" is what most of Thunderbird is. (If you exclude all of Gecko, which is part of Firefox, maybe this isn't true...)

I wonder if we can/should just leave this field blank.

Comment (afranke) [reply gatsby/content/projects/2017-05-02-thunderbird.mdx:9]: I’m guessing the point of this field is “if I wanted to contribute to *this*, which language would I need to know/learn”, where this is “this matrix related thing”. In this context, as I understand it, JavaScript only is relevant.

Comment (clokep) [reply gatsby/content/projects/2017-05-02-thunderbird.mdx:9]: Ah, yeah from the POV of a contributor having this field makes sense. From the POV of a user, not so much!

File: gatsby/content/projects/2017-05-02-thunderbird.mdx (hunk 3 above)
Comment (clokep) [reply gatsby/content/projects/2017-05-02-thunderbird.mdx]: I rephrased to "beta and release".

File: gatsby/content/projects/2017-05-02-thunderbird.mdx (hunk 2 above)
Comment (clokep) [reply gatsby/content/projects/2017-05-02-thunderbird.mdx:18]: I can definitely add more, I'll try to see if there's a full list somewhere and not just leave them out?

Comment (afranke) [reply gatsby/content/projects/2017-05-02-thunderbird.mdx:18]: [`src/pages/clients-matrix.js`](https://github.com/matrix-org/matrix.org/blob/33914ac58515b2e26af8402aab8c3aef13a3eaeb/gatsby/src/pages/clients-matrix.js#L225) has the full list.

I reckon having the full list in a file with values as `no` makes it easier to update them at a later point when one reopens the file and notices the feature is now available.

Comment (clokep) [reply gatsby/content/projects/2017-05-02-thunderbird.mdx:18]: Thanks. I've added those. I'll see about moving it / getting some screenshots.

Comment (benparsons) [reply gatsby/content/projects/2017-05-02-thunderbird.mdx:18]: `/clients` contains clients with `featured: true` in their yaml. Please move the file to the clients subdirectory as they are updated (can remove the date from the filename too)

**Comment by clokep** [comment]
I think the last thing to do here is to dig up a screenshot.

**Comment by afranke** [comment]
We did a category cleanup and the thunderbird file was moved as part of this, resulting in a conflict. Do you mind resolving it?

How is that screenshot coming along?

File: gatsby/content/projects/clients/thunderbird.mdx [hunk 5]
```diff
@@ -0,0 +1,57 @@
+---
+layout: projectimage
+title: Mozilla Thunderbird
+categories: 
+ - client
+description: A free open-source email, calendar & chat app
+author: Mozilla
+maturity: Beta
+language: JavaScript
```
Comment (freaktechnik) [inline gatsby/content/projects/clients/thunderbird.mdx:9]: should probably also declare the SDK we use.

```suggestion
language: JavaScript
SDK: matrix-js-sdk
```

**Comment by clokep** [comment]
I unfortunately haven't been able to get a good screenshot here since my account currently seems to be causing issues for Thunderbird. Do we think we could merge this without it and add one in the future once this is out of beta? Or if not, maybe @freaktechnik could get one?

**Comment by afranke** [comment]
Ok, let’s roll with that. Feel free to open a new PR for the screenshot, whenever you manage to get one.

**Pull request merged**